Formula/vmdktool.rb

```diff
@@ -0,0 +1,21 @@
+class Vmdktool < Formula
+  desc "Converts raw filesystems to VMDK files and vice versa"
+  homepage "http://manned.org/vmdktool"
+  url "https://people.freebsd.org/~brian/vmdktool/vmdktool-1.4.tar.gz"
+  sha256 "981eb43d3db172144f2344886040424ef525e15c85f84023a7502b238aa7b89c"
+
+  def install
+    system "make", "CFLAGS='-D_GNU_SOURCE -g -O -pipe'"
+    bin.mkpath
```

Comment:
Sorry, ignore me. I didn't realize this was prefixed inside the cellar; you'll need those `mkpath`s after all.